ompi/mca/mtl/ofi/mtl_ofi_types.h

```diff
@@ -59,6 +59,7 @@ typedef struct mca_mtl_ofi_module_t {
     int enable_sep;                     /* MCA to enable/disable SEP feature */
     int thread_grouping;                /* MCA for thread grouping feature */
     int num_ofi_contexts;               /* MCA for number of contexts to use */
+    bool disable_hmem;                   /* MCA to enable/disable request for FI_HMEM support from provider */
```

Comment:
Curious about a difference between mtl/ofi and btl/ofi.

For mtl/ofi I see the mca parameters are registered under `struct mca_mtl_ofi_module_t`, but btl/ofi uses `mca_btl_ofi_component_t`. Is there a reason for this?

Reply:
Not that i'm aware of.  good point though i'll double check about which of the struct types to use for these booleans.

Reply:
it looks a bit arbitrary looking in different frameworks/components when to put fields in the mca_XXX_component_t vs mca_XXX_module_t (seems like sometimes the module string is left out).  I'd prefer to keep with the pattern in each of the ofi components with this PR.